lib/rhc/commands/app.rb

```diff
@@ -425,7 +425,8 @@ def show(app_name)
         case options.gears
         when 'quota'
           opts = {:as => :gear, :split_cells_on => /\s*\t/, :header => ['Gear', 'Cartridges', 'Used', 'Limit'], :align => [nil, nil, :right, :right]}
-          table_from_gears('echo "$(du --block-size=1 -s 2>/dev/null | cut -f 1)"', groups, opts) do |gear, data, group|
+          # quota 'blocks' are actually kb, so have to multiply by 1024, see 'man quota'
+          table_from_gears('echo "$(quota | egrep -v \'quotas|Filesystem\' | awk \'{ print $2 * 1024 }\')"', groups, opts) do |gear, data, group|
```

Comment:
@Miciah 
- replaced the egrep with NR = 3 :)
- from command line, no need for 'echo "$(foo)" but this format is the only one that works in this case, I've tried everything! I can investigate further into why, but that's the way the du cmd is currently, maybe for the same unknown reason.
- I agree the blocksize shouldn't be hardcoded.  'cat /usr/include/sys/mount.h | grep BLOCK_SIZE | awk ' NR=1 {print $3}' also works but it's so far been tricky to get this value into the quota cmd.  There must be a way, I'll figure it out (unless you know?)
thanks!